src/Features/CSharp/Portable/Completion/CompletionProviders/ImportCompletion/ImportCompletionProviderHelper.cs

```diff
@@ -25,6 +25,23 @@ private record class CacheEntry(DocumentId? GlobalUsingsDocumentId, Checksum Glo
 
         public static async Task<ImmutableArray<string>> GetImportedNamespacesAsync(SyntaxContext context, CancellationToken cancellationToken)
         {
+            var scopes = context.SemanticModel.GetImportScopes(context.Position, cancellationToken);
```

Comment:
can you just pass 0 here to just get the global usings added to the file?

Reply:
This helper returns all usings, not just global ones, so we have to look through all scopes

Reply:
That seems reasonable?